docs/howto/custom-management-commands.txt

```diff
@@ -62,12 +62,12 @@ look like this::
                 try:
                     poll = Poll.objects.get(pk=poll_id)
                 except Poll.DoesNotExist:
-                    raise CommandError('Poll "%s" does not exist' % poll_id)
+                    raise CommandError(self.style.ERROR, 'Poll "%s" does not exist' % poll_id)
 
                 poll.opened = False
                 poll.save()
 
-                self.stdout.write('Successfully closed poll "%s"' % poll_id)
+                self.stdout.write(self.style.SUCCESS, 'Successfully closed poll "%s"' % poll_id)
```

Comment:
Looks like the syntax is actually `self.style.SUCCESS('Successfully closed poll "%s"' % poll_id)`